app/src/main/java/com/example/hulk/mtindo/Createstore.java

```diff
@@ -218,14 +225,17 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (resultCode == RESULT_OK) {
             if (requestCode == SELECT_PICTURE) {
                 Uri selectedImageUri = data.getData();
+                Log.i(TAG, "Image Uri : " + selectedImageUri);
                 selectedImagePath = getPath(selectedImageUri);
-                System.out.println("Image Path : " + selectedImagePath);
+                Log.i(TAG, "Image Path : " + selectedImagePath);
                 image.setImageURI(selectedImageUri);
-                BitmapFactory.Options options = new BitmapFactory.Options();
-                options.inPreferredConfig = Bitmap.Config.ARGB_8888;
-                Bitmap bitmap = BitmapFactory.decodeFile(selectedImagePath, options);
-                image.setImageBitmap(bitmap);
 
+                try {
+                     image.setImageBitmap(MediaStore.Images.Media.getBitmap(this.getContentResolver(), selectedImageUri));
```

Comment:
replace this line with the following two lines; that should fix the problem:

bitmapImage = MediaStore.Images.Media.getBitmap(this.getContentResolver(), selectedImageUri);
image.setImageBitmap(bitmapImage);